nixos/modules/services/networking/consul.nix

```
@@ -99,6 +99,7 @@ in
 
       extraConfig = mkOption {
         default = { };
+        type = types.lines;
```

I don't think this is the right type.

```suggestion
        type = types.attrsOf types.str;
```